message_definitions/v1.0/ras_a.xml

```diff
@@ -10,7 +10,140 @@
+        <param index="6" label="Position Y or Longitude">Y or Longitude (WGS84) coordinate of the portal position. NaN or INT32_MAX if unknown (or if using param 1 to set the portal).</param>
+        <param index="7" label="Position Z or Altitude (MSL)">Z or Altitude (MSL) coordinate of the portal position. NaN when unknown or when using param 1 to set the portal.</param>
+      </entry>
+      <entry value="40501" name="MAV_CMD_SET_INGRESS_PORTAL" hasLocation="true" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Sets a specific portal to be an entry portal. Defines also the approach vector for a vehicle to approach and go through the portal.
+          COMMAND_INT should be used so to set the MAV_FRAME and consequently, frame and coordinates of the approach vector (MAV_FRAME_GLOBAL means global coordinates are being used,
+          while MAV_FRAME_LOCAL_NED means local NED coordinates are being used).
+        </description>
+        <param index="1" label="Portal ID" minValue="0" maxValue="4294967295" increment="1">ID of the portal to be set as an ingress point.</param>
+        <param index="2" label="Approach vector initial point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="3" label="Approach vector initial point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="4" label="Approach vector initial point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the initial point of the approach vector to the portal. NaN if unknown.</param>
+        <param index="5" label="Approach vector final point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="6" label="Approach vector final point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX if unknown.</param>
+        <param index="7" label="Approach vector final point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the final point of the approach vector to the portal. NaN if unknown.</param>
+      </entry>
+      <entry value="40502" name="MAV_CMD_SET_EGRESS_PORTAL" hasLocation="true" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Sets a specific portal to be an entry portal. Defines also the approach vector for a vehicle to approach and go through the portal.
```

Comment:
Should be "exit portal"

Reply:
Well, the portals should be consistently named. Ingress, Egress or ENTRY/EXIT. I prefer the latter.